app/forms/field_handlers/field_handler.py

```diff
@@ -47,12 +48,12 @@ def label(self) -> Optional[str]:
     def guidance(self) -> str:
         return self.answer_schema.get("guidance", "")
 
-    def get_validation_message(self, message_key: str) -> Optional[str]:
+    def get_validation_message(self, message_key: str) -> Union[str, Any]:
         return self.validation_messages.get(message_key) or self.error_messages.get(
             message_key
         )
```

Comment:
i tried that but it break whole loads of tests in `field handler` because of this piece of code in here    https://github.com/ONSdigital/eq-questionnaire-runner/blob/877ca2e025e56f6d59cee2e868cf4d611e695036/app/forms/field_handlers/number_handler.py#L90
with error as ` AttributeError: 'NoneType' object has no attribute 'copy'
as it is inherited from.`    
`FieldHandler`  class

Reply:
That will be because the test will be passing `None` when it no longer should. They will need updating to pass an mapping in.